core/inject.js

```diff
@@ -55,9 +56,16 @@ Blockly.inject = function(container, opt_options) {
   var subContainer = goog.dom.createDom('div', 'injectionDiv');
   container.appendChild(subContainer);
   var svg = Blockly.createDom_(subContainer, options);
+
+  // Create surfaces for dragging things. These are optimizations
+  // so that the broowser does not repaint during the drag.
   var blockDragSurface = new Blockly.BlockDragSurfaceSvg(subContainer);
   blockDragSurface.createDom();
-  var workspace = Blockly.createMainWorkspace_(svg, options, blockDragSurface);
+  var workspaceDragSurface = new Blockly.workspaceDragSurfaceSvg(subContainer);
+  workspaceDragSurface.createDom();
```

Comment:
Would you ever call the workspace drag surface constructor and not immediately call createDom?  If not, the createDom call should be in the constructor.